Packages.props

```diff
@@ -0,0 +1,108 @@
+    <PackageReference Update="Microsoft.Azure.Amqp" Version="[2.4.2, 3.0.0)" />
+    <PackageReference Update="Microsoft.Azure.Batch" Version="[10.0.0, 11.0.0)" />
+    <PackageReference Update="Microsoft.Azure.Graph.RBAC" Version="2.2.2-preview" />
+    <PackageReference Update="Microsoft.Azure.KeyVault.Core" Version="3.0.3" />
+    <PackageReference Update="Microsoft.Azure.Management.Batch" Version="4.2.0" />
+    <PackageReference Update="Microsoft.Azure.Management.ContainerRegistry" Version="2.0.0" />
+    <PackageReference Update="Microsoft.Azure.Management.EventGrid" Version="4.0.1-preview" />
+    <PackageReference Update="Microsoft.Azure.Management.HDInsight" Version="4.1.0-preview" />
+    <PackageReference Update="Microsoft.Azure.Management.ResourceManager" Version="[1.6.0-preview, 2.0.0)" />
+    <PackageReference Update="Microsoft.Azure.Management.Sql" Version="1.22.0-preview" />
+    <PackageReference Update="Microsoft.Azure.Management.Storage" Version="6.4.0-preview" />
+    <PackageReference Update="Microsoft.Azure.ResourceManager" Version="[1.1.0-preview]" />
+    <PackageReference Update="Microsoft.Azure.Services.AppAuthentication" Version="[1.0.3, 2.0.0)" />
+    <PackageReference Update="Microsoft.Azure.Storage.Blob" Version="9.4.2" />
+    <PackageReference Update="Microsoft.Azure.Test.HttpRecorder" Version="[1.13.1, 2.0.0)" />
+    <PackageReference Update="Microsoft.Bcl.AsyncInterfaces" Version="1.0.0-preview6.19259.10" />
+    <PackageReference Update="Microsoft.Bcl.Json.Sources" Version="4.6.0-preview3.19128.7" />
+    <PackageReference Update="Microsoft.Build.Tasks.Core" Version="15.5.180" />
+    <PackageReference Update="Microsoft.Build" Version="15.5.180" />
+    <PackageReference Update="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="2.6.1" />
+    <PackageReference Update="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.6.2" />
+    <PackageReference Update="Microsoft.CodeAnalysis" Version="2.3.0" />
+    <PackageReference Update="Microsoft.Extensions.Configuration" Version="1.0.2" />
+    <PackageReference Update="Microsoft.Extensions.Configuration.Abstractions" Version="2.2.0" />
+    <PackageReference Update="Microsoft.Extensions.Configuration.Binder" Version="2.2.0" />
+    <PackageReference Update="Microsoft.Extensions.Configuration.Json" Version="1.0.2" />
+    <PackageReference Update="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.2.0" />
+    <PackageReference Update="Microsoft.Extensions.DependencyInjection" Version="2.2.0" />
+    <PackageReference Update="Microsoft.Extensions.Logging.Abstractions" Version="2.2.0" />
+    <PackageReference Update="Microsoft.Extensions.Options" Version="2.2.0" />
+    <PackageReference Update="Microsoft.Extensions.PlatformAbstractions" Version="1.1.0" />
+    <!-- <PackageReference Update="Microsoft.IdentityModel.Clients.ActiveDirectory" Version="3.14.0" /> -->
```

Comment:
Let's try aligning everyone to the most common version which appears to be 4.5.1.